envisage/tests/test_slice.py

```diff
@@ -78,16 +76,12 @@ class Foo(HasTraits):
         self.f = Foo(l=TEST_LIST)
         self.f.on_trait_change(listener, 'l_items')
 
-        return
-
     def tearDown(self):
         """ Called immediately after each test method has been called. """
 
         # Make sure we successfully recreated the operation.
         self.assertEqual(self.f.l, listener.clone)
```

Comment:
Making test assertions in the `tearDown` method seems a little icky. I don't think you need to address/modify it in this PR though.

Reply:
Let's fix it, while we're here.